backend/user_management/minio/config.py

```
@@ -0,0 +1,22 @@
+import os
+from minio import Minio
+from dotenv import load_dotenv
+
+# Load environment variables from .env file
+load_dotenv()
+
+MINIO_CLIENT = Minio(
+    endpoint="localhost:9000",  # MinIO URL
+    access_key=os.getenv("MINIO_ACCESS_KEY"), 
+    secret_key=os.getenv("MINIO_SECRET_KEY"),
+    secure=False  # Set to True if using HTTPS
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)